### PR TITLE
Cases pagination, sorting, filtering, and search

### PR DESCRIPTION
**Pull request by MatanYadaev**

### CODE REVIEW

File: apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx [hunk 1]
```diff
@@ -53,10 +53,10 @@ export const Navbar: FunctionComponent = () => {
       <ul className={`menu menu-compact w-full space-y-2`}>
         {filters?.map(({ id, name, entity }) => (
           <NavItem
-            href={`/en/case-management/entities?entity=${entity}&filterId=${id}&filterName=${name}`}
+            href={`/en/case-management/entities?entity=${entity}&filterId=${id}`}
             key={id}
             className={ctw(`capitalize`, {
-              'bg-muted font-bold': name === searchParams?.filterName,
+              'bg-muted font-bold': id === searchParams?.filterId,
```
Comment (MatanYadaev) [inline apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx:59]: Unrelated fix:
We added the `filterName` to the query params just for comparing the current filter. It was unnecessary because we already have `filterId`.

File: apps/backoffice-v2/src/common/hooks/useSearchParamsByEntity/validation-schemas.ts [hunk 2]
```diff
@@ -3,11 +3,10 @@ import { CaseStatus, CaseStatuses, States } from '../../enums';
 
 export const SearchSchema = z.object({
   sortDir: z.enum(['asc', 'desc']).catch('desc'),
-  pageSize: z.coerce.number().int().positive().catch(10),
+  limit: z.coerce.number().int().positive().catch(50),
```
Comment (MatanYadaev) [inline apps/backoffice-v2/src/common/hooks/useSearchParamsByEntity/validation-schemas.ts:6]: Increased the page size because the search is in the client only, so at least it searches a big amount of cases.

File: services/workflows-service/src/prisma/prisma.service.ts [hunk 3]
```diff
@@ -3,6 +3,12 @@ import { PrismaClient } from '@prisma/client';
 
 @Injectable()
 export class PrismaService extends PrismaClient implements OnModuleInit {
+  constructor() {
+    super({
+      errorFormat: 'pretty',
```
Comment (MatanYadaev) [inline services/workflows-service/src/prisma/prisma.service.ts:8]: It will add colors to the Prisma errors.

File: services/workflows-service/src/workflow/utils/toPrismaOrderBy.ts [hunk 4]
```diff
@@ -21,23 +21,23 @@ export const toPrismaOrderBy = <
...
+      } as { [K in TColumn]: TDirection },
     };
   }
 
   return {
-    [column]: direction,
+    business: {
+      [column]: direction,
+    },
   };
```
Comment (MatanYadaev) [inline services/workflows-service/src/workflow/utils/toPrismaOrderBy.ts:42]: Same logic, just a bit more readable.

File: services/workflows-service/src/workflow/utils/toPrismaWhere.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { Prisma, WorkflowRuntimeDataStatus } from '@prisma/client';
+
+type Filters = {
+  assigneeId?: (string | null)[];
+  status?: WorkflowRuntimeDataStatus[];
+};
+
+export const toPrismaWhere = (filters: Filters): Prisma.WorkflowRuntimeDataWhereInput => {
```
Comment (MatanYadaev) [inline services/workflows-service/src/workflow/utils/toPrismaWhere.ts:8]: This function, and the `toPrismaOrderBy`, are workflow-specific. They won't fit for users, etc.
Should I rename them or place them in another folder to make it clearer?

**Review by alonp99 — APPROVED** [review]
